data/locale.yml

```diff
@@ -14,6 +14,14 @@ nav:
   search: Search
   userland: Userland
 
+keyboard_modal:
+  focus_search: Focus search bar.
+  focus_and_clear: Focus search bar and clearns it.
+  scroll_down: Scroll the searched result bottom.
+  scroll_up: Scroll the searched result upper.
+  open: Opens the searched result.
+  open_new_tab: Opens the searched result in new tab.
+
```

Comment:
Some ideas for these phrases:

* Focus search bar
* Focus and clear search bar
* Select the next search result
* Select the previous search result
* Open the selected search result
* Open the selected search result in a new tab